src/SDKs/PowerBIDedicated/PowerBIDedicated.Tests/PowerBIDedicated.Tests.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />  
   <PropertyGroup>
     <PackageId>PowerBIDedicated.Tests</PackageId>
-    <VersionPrefix>0.9.0-preview</VersionPrefix>
```

Comment:
Test project version must always be set to `1.0.0`

Reply:
Like pointed out earlier, this must be set to `1.0.0`, not `1.0.0-preview`
The `AssemblyInfo.cs` for the test project must also be set to `1.0.0`